PayPal/src/main/java/com/braintreepayments/api/paypal/vaultedit/PayPalVaultEditRequest.kt

```diff
@@ -12,7 +12,8 @@ import com.braintreepayments.api.core.ExperimentalBetaApi
 @ExperimentalBetaApi
 open class PayPalVaultEditRequest
 @JvmOverloads constructor(
-    open val editPayPalVaultId: String
+    open val editPayPalVaultId: String,
+    open val hasUserLocationConsent: Boolean
```

Comment:
```suggestion
    open val hasUserLocationConsent: Boolean,
    open val editPayPalVaultId: String
```

Nitpick because this is how the integration guide has it and aligns with the `PayPalRequest` parameter order